apps/ledger-live-desktop/src/renderer/modals/Receive/steps/StepReceiveFunds.tsx

```diff
@@ -103,6 +104,11 @@ const Receive1ShareAddress = ({
           <UTXOAddressAlert />
         </Box>
       )}
+      <FeatureToggle featureId="lldMemoTag">
```

Comment:
Kevin and Anthony suggested to only show the memos/tags message for coins that might need them. So I added a list in LLD: https://github.com/LedgerHQ/ledger-live/pull/7989/files#diff-95f10ef90834180737095894a09612d65e36dc5ec286fb07a2ed743b3cede50e

I'm a bit worried that we might miss some of them (especially on the send flow). What do you think of this ?